rhel7/profiles/e8.profile

```diff
@@ -0,0 +1,132 @@
+documentation_complete: true
+
+title: 'Australian Cyber Security Centre (ACSC) Essential Eight System Hardening for Red Hat Enterprise Linux 7'
```

Comment:
@shaneboulden can you remove 'for Red Hat Enterprise Linux 7' and add it to the description?

Reply:
Done, and shortened the title to reflect other profiles